src/main/java/ita/softserve/course_evaluation/CourseEvaluationApplication.java

```diff
@@ -10,4 +15,14 @@ public static void main(String[] args) {
 		SpringApplication.run(CourseEvaluationApplication.class, args);
 	}
 	
+	@Bean
+	public ModelMapper modelMapper() {
```

Comment:
I suggest creating a configuration class for this

Reply:
i moved it to -> /configuration/ApplicationConfig.java